app/src/routes/definitions.json

```diff
@@ -14,9 +14,19 @@
         "resetPassword": "/resetPassword",
         "signUp": "/signup"
     },
-    "canvasEditor": "/canvas/editor",
-    "canvasEmbed": "/canvas/embed/:id",
-    "canvases": "/core/canvases",
+    "canvases": {
+        "edit": "/canvas/editor/:id?",
```

Comment:
Would be nice to rename these at one point to just `/canvases/...` or `/dashboards/...` instead of the singular form. I'd also prefer the `/resource/:id/specifier` type of url, eg. `/canvases/:id/edit` or `/canvases/:id/embed`. But beyond the scope here